Assets/Scripts/StarComponent.cs

```diff
@@ -45,6 +45,7 @@ void OnMouseDown()
         if (!EventSystem.current.IsPointerOverGameObject() && dataController && dataController.starInfoPanel)
         {
             dataController.ChangeConstellationHighlight(starData.Constellation);
+            Debug.Log("(BD=" + starData.BayerDesignation + "),(FD=" + starData.FlamsteedDesignation + "),(ConstShort=" + starData.Constellation + "),(ConstFull=" + starData.ConstellationFullName + ")");

```

Comment:
I think this looks to be pulling the right data, I'd go ahead and put it in the panels next so end users can see it too

Reply:
Yeah, that's the plan.  I just want the import and the UI changes in different PRs so they are easier to review.